### PR TITLE
Feature/account page improvement

### PR DESCRIPTION
**Pull request by vipin-webkul**
Added feature to update Ui-locale and timezone in the account edit page.

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/account/edit.blade.php [hunk 1]
```diff
@@ -94,19 +89,71 @@ class="primary-button"
...
+
+                        <x-admin::form.control-group.control
+                            type="select"
+                            id="ui_locale_id"
+                            name="ui_locale_id"
+                            rules="required"
+                            :value="old('ui_locale_id') ?: $user->ui_locale_id"
+                            :label="trans('admin::app.account.edit.ui-locale')"
+                            :placeholder="trans('admin::app.account.edit.ui-locale')"
+                            :options="$locales"
```
Comment (devansh-pal-webkul) [inline packages/Webkul/Admin/src/Resources/views/account/edit.blade.php]: The options can be added like this without the need of an extra variable:
```php
:options="core()->getAllActiveLocales()"
```

File: packages/Webkul/Admin/src/Resources/views/account/edit.blade.php [hunk 2]
```diff
@@ -94,19 +89,71 @@ class="primary-button"
...
+                        @endphp
+
+                        <x-admin::form.control-group.control
+                            type="select"
+                            name="timezone"
+                            rules="required"
+                            :value="old('timezone') ?: $user->timezone"
+                            :label="trans('admin::app.account.edit.user-timezone')"
+                            :placeholder="trans('admin::app.account.edit.user-timezone')"
+                            :options="$timezones"
```
Comment (devansh-pal-webkul) [inline packages/Webkul/Admin/src/Resources/views/account/edit.blade.php]: Here too the options can be added like this without the need of an extra variable:
```php
:options="json_encode(core()->getTimeZones())"
```

File: packages/Webkul/User/tests/Feature/UserTest.php [hunk 3]
```diff
@@ -201,3 +201,204 @@
...
+    get(route('admin.account.edit'))
+        ->assertStatus(200)
+        ->assertSeeText(trans('admin::app.account.edit.title'));
+});
+
+it('should give validation errors when updating the user', function () {
+    $this->loginAsAdmin();
+
+    $currentPass = Hash::make('password');
+    $admin = Admin::factory()->create([
```
Comment (devansh-pal-webkul) [inline packages/Webkul/User/tests/Feature/UserTest.php]: Remove this from all the test cases as it is not being used.

File: packages/Webkul/User/tests/Feature/UserTest.php [hunk 4]
```diff
@@ -201,3 +201,204 @@
...
+        'current_password'      => 'password',
+        'password'              => 'password2',
+        'image'                 => '',
+        'timezone'              => 'Asia/Kolkata',
+        'ui_locale_id'          => 2,
+        'password_confirmation' => 'password2',
+    ]);
+
+    $response->assertStatus(302);
+    $response->assertRedirect();
```
Comment (devansh-pal-webkul) [inline packages/Webkul/User/tests/Feature/UserTest.php:332]: Both are same only need to use one assertRedirect checks for the status code 302 also.
```php
    $response->assertRedirect();
```

**Review by devansh-pal-webkul — CHANGES_REQUESTED** [review]
@vipin-webkul, please make the requested changes.

**Comment by vipin-webkul** [comment]
@devansh-pal-webkul 
I have updated the code as mentioned.
Please check.

**Pull request merged**